src/server/managers/AppListenerManager.ts

```diff
@@ -63,8 +116,18 @@ export class AppListenerManager {
         return results;
     }
 
+    public isEventBlocked(event: AppInterface): boolean {
+        const blockedEventList = this.blockedEvents.get(event);
+
+        return !!(blockedEventList && blockedEventList.length);
```

Comment:
```suggestion
        return !!(blockedEventList?.length);
```

Reply:
Our TypeScript version does not support optional chaining :)